Hibernate/OneToOneMapping/DeleteDemo.java

```diff
@@ -0,0 +1,38 @@
+package OneToOneMapping;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class DeleteDemo {
+
+	public static void main(String[] args) {
+		SessionFactory factory = new Configuration()
+								.configure("hibernate.cfg.xml")
+								.addAnnotatedClass(Instructor.class)
+								.addAnnotatedClass(InstructorDetail.class)
+								.buildSessionFactory();
+		
+		Session session = factory.getCurrentSession();
+		try {			
+			session.beginTransaction();
+			int theId = 1;
+			Instructor tempInstructor = 
+					session.get(Instructor.class, theId);
+			
+			System.out.println("Found instructor: " + tempInstructor);
+			if (tempInstructor != null) {
+			
+				System.out.println("Deleting: " + tempInstructor);
+				session.delete(tempInstructor);				
+			}
+			session.getTransaction().commit();
+			
+			System.out.println("Done!");
```

Comment:
See the Logger class instead of System.out.println()
Something like Logger loggger = LoggerFactory.getLogger(this.getClass()) should do

Reply:
Loggers are basically used in place of System.out.println() for may uses ike we can maintain logs in a file in realtime application and also we can customize what information is to be printed along with the actual message.